app/modules/import-export-customization/assets/js/import/pages/import-kit.js

```diff
@@ -44,6 +45,10 @@ export default function ImportKit() {
 		}
 	}, [ id, referrer, fileUrl, nonce, dispatch ] );
 
+	useEffect( () => {
+		AppsEventTracking.sendPageViewsWebsiteTemplates( elementorCommon.eventsManager.config.secondaryLocations.kitLibrary.kitImportUploadBox );
```

Comment:
Does this and `app/modules/import-export-customization/assets/js/import/pages/import-customization.js` have the same event?